opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
Made this comment somewhere else, but I kindof like google-cloud-tracing's SpanKind for annotating the "side" of the span (no pun intended).

peer is ok as it solves the need for a separate namespace for client vs server. The tradeoff is that some folks will feel the word peer less clear than just saying server. Ex. consider those writing client libraries... they only talk about the server and visa versa. Using peer optimizes for those writing both sides (ex things like finagle, grpc, etc), which I suspect is a subset of the total audience of the api.